### PR TITLE
MAIN - pr preview

### PR DESCRIPTION
**Pull request by paschalidi**
Happy holidays @VivyTeam/web ! 

I thought that gifs are cool but previewing our PRs is even ✨✨cooler✨✨. 

I am fun of [`surge`](https://surge.sh/) these days and wanted to try it out. Its super cool for deploying static websites.
When I say "super cool" I mean easy to use 😎 

Have a look and let me know what you think!
Also since I didnt communicate this with anybody its also fine if you say we that this is not something we would like to have. I will then close the PR and no big problem 👍

### CODE REVIEW

**Comment by paschalidi** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>Missing reviewer - <i>Looks like this pull request is missing a reviewer. That's OK as long as this PR is still a 'work in progress', if that is the case please add 'WIP' in the PR title or corresponding label.</i></td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>✅  Get a 🏆 for adding description on this PR. - <i>Celebrate on every step!</i></td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 91311054bc87c80eb452c20907c73bb2540e8f28
</p>

File: .circleci/config.yml [hunk 1]
```diff
@@ -33,6 +33,31 @@ jobs:
...
+            - v1-dependencies-{{ checksum "package.json" }}
+            # fallback to using the latest cache if no exact match is found
+            - v1-dependencies-
+      # Deploy PR-Preview
+      - run:
+          name: PR Preview Storybook
+          command: |
+            npm install
+            npm run build:static
+            yarn run surge --project storybook-static --domain ${CIRCLE_BRANCH}-${CIRCLE_PROJECT_USERNAME}-${CIRCLE_PROJECT_REPONAME}.surge.sh
```
Comment (lanwen) [inline .circleci/config.yml]: I think should be other way around - branch-repo-username

**Comment by paschalidi** [comment]
@lanwen @Weetbix please have another look! Now the url that is being generated will be ${CIRCLE_BUILD_NUM}-${CIRCLE_PROJECT_REPONAME}-${CIRCLE_PROJECT_USERNAME}.surge.sh

* CIRCLE_BUILD_NUM : The number of the CircleCI build.
* CIRCLE_PROJECT_REPONAME: The name of the repository of the current project.
* CIRCLE_PROJECT_USERNAME: The organisation name. 

For instance [1828-vivy-components-vivyteam.surge.sh](http://1828-vivy-components-vivyteam.surge.sh)

As we discussed with Kirill in private the idea was to use branch names as part of the url. However branch names can cause the pr-preview surge command to fail when `/` is part of the branch name. I couldnt find a way that in yml files we can replace that part of the string. if one of you can find one I am happy to implement it.

Please let me know your thoughts and feedback on this!

**Review by lanwen — APPROVED** [review]

**Review by Weetbix — APPROVED** [review]
Awesome! 😍

**Pull request merged**